content/blog/2020-03-dplyr-1-0-0-is-coming-soon/index.Rmarkdown

```diff
@@ -0,0 +1,209 @@
+  to call upon, it is likely that some new bugs have will have slipped in. One
+  goal of this blog post is to encourage the adventurous to try out dplyr today 
+  and report bugs so those who wait until the CRAN release won't see them.
+
+* There may be some performance decreases. We've put in quite a lot of time 
+  to ensure most performance changes are positive or only slightly negative,
+  but we may have missed some cases. Please let us know if you notice dplyr 
+  code that is substantially slower in this version.
+
+Overall, we believe that the upsides to using vctrs outweigh the downsides, particularly in the long term as we use vctrs to power more and more of the tidyverse. There will be pain, but we hope to keep it as small as possible by ripping the band aid off quickly. After this release, dplyr will be a 1.0.0, which means that you should expect very few breaking changes in the future. We'll continue to add new functions and arguments but will be much more conservative about modifying or removing features.
+
+If the potential for changes makes you nervous, now is a good time to learn about [renv](https://rstudio.github.io/renv/). renv allows to create isolated, reproducible, projects so that you can experiment with new package versions while feeling secure that your existing projects will continue to work as they always have.
+
+## Lifecycle
+
+To make the transition to dplyr 1.0.0 easier we have invested a lot of time in clarifying where functions (and some arguments) fall in their lifecycle. Our goal is to better inform you of our thinking about functions. You won't always agree with our decisions, but you shouldn't be surprised!
+
+There are three stages in the lifecycle that are particularly important to know about in dplyr 1.0.0:
+
+* **Deprecated** functions (![deprecated badge](lifecycle-deprecated.svg)) 
+  are on their way out and you'll should replace them with their modern 
+  alternatives in the near future.
+  
+* **Superseded** functions (![superseded badge](lifecycle-superseded.svg)) 
+  aren't going away, but we no longer recommend using them because we think 
+  we've discovered better alternatives. There's no rush, but we suggest that 
+  you learn about their replacements and phase out your use of the superseded 
+  functions over the next year or two.
+
+* **Experimental** features (![experimental badge](lifecycle-experimental.svg)) 
+  are those features that we're cautiously optmistic about, but want to get 
+  more feedback on before we fully commit to them. Please try them out and let 
+  us know what you think!
+
+The following sections describe each stage in more detail, illustrated with the most important functions in that stage in dplyr 1.0.0
+
+### Deprecated functions
+
+Deprecation is the most visible of the lifecycle stages because you're forced to immediately confront it when you use a deprecated function:
+
+```{r}
+mtcars2 <- add_rownames(mtcars)
```

Comment:
The message here is a bit surprising. In the preview I see this:

```
mtcars2 <- add_rownames(mtcars)
#> Warning: `tbl_df()` is deprecated as of dplyr 1.0.0.
#> Please use `tibble::rownames_to_column()` instead.
```

What's up with `tbl_df()`?

Reply:
Oooops, good catch!